docs/chaos/behavior.md

```diff
@@ -0,0 +1,126 @@
+# Behavior monkey strategy
+
+## About
+
+- **Options**: [`BehaviorStrategyOptions`](xref:Polly.Simmy.Behavior.BehaviorStrategyOptions)
+- **Extensions**: `AddChaosBehavior`
+- **Strategy Type**: Proactive
+
+---
+
+The behavior chaos strategy is designed to inject custom behaviors into system operations right before such an operation is invoked. This strategy is flexible, allowing users to define specific behaviors such as altering the input, simulating resource exhaustion, putting the system in a given state before the actual operation is called, or other operational variations to simulate real-world scenarios.
+
+## Usage
+
+<!-- snippet: Behavior -->
```

Comment:
These snippets should all have accompanying C# code in the [Snippets project](https://github.com/App-vNext/Polly/tree/main/src/Snippets) that validates they compile.

Reply:
also the snippets should be prefixed with `simmy` or `chaos`, i.e. `simmy-behavior`

Reply:
Done